.github/workflows/pull-request.yml

```diff
@@ -3,73 +3,7 @@ name: Ballerina HTTP module build
 on: [pull_request]
 
 jobs:
-  ubuntu-build:
-    name: Build on Ubuntu
-    runs-on: ubuntu-latest
-    timeout-minutes: 60
-    concurrency:
-      group: ${{ github.head_ref }}-ubuntu
-      cancel-in-progress: true
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ github.actor }}
-          packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-            ./gradlew build
-      - name: Generate CodeCov Report
-        if: github.event_name == 'pull_request'
-        uses: codecov/codecov-action@v3
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-
-  windows-build:
-    name: Build on Windows
-    runs-on: windows-latest
-    timeout-minutes: 60
-    concurrency:
-      group: ${{ github.head_ref }}-windows
-      cancel-in-progress: true
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ github.actor }}
-          packagePAT: ${{ secrets.GITHUB_TOKEN }}
-          JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8
-        run: |
-            ./gradlew.bat build
-
-  ubuntu-build-without-native-tests:
-      name: Build on Ubuntu without native tests
-      timeout-minutes: 60
-      concurrency:
-          group: ${{ github.head_ref }}-ubuntu-without-native
-          cancel-in-progress: true
-      runs-on: ubuntu-latest
-      steps:
-          - name: Checkout Repository
-            uses: actions/checkout@v3
-          - name: Set up JDK 17
-            uses: actions/setup-java@v3
-            with:
-                distribution: 'temurin'
-                java-version: 17.0.7
-          - name: Build with Gradle
-            env:
-                packageUser: ${{ github.actor }}
-                packagePAT: ${{ secrets.GITHUB_TOKEN }}
-            run: ./gradlew build -x http-native:test
+  call_workflow:
```

Comment:
Don't we have a to skip `-x http-native:test` here ?

Reply:
We have an additional step to run Ubuntu tests without native tests.
How to we handle it here? Should I add another job to that?

Reply:
Ok. If there is no need to separate this step in to two, We don't have to add the skip flag here.

Reply:
Ack, so we will be having two builds and there will be no `Build on Ubuntu without native tests` right?